Form-Controls/index.html

```diff
@@ -1,28 +1,129 @@
+      <li><a href="#">Woman</a></li> 
+      <li><a href="#">Kids</a></li> 
+    </ul>
+  </div>
+  </nav>
+  <hr class="divider"> </hr>
+   <!--end nav menu-->
+
+ <!--HEADER-->
   <header>
-    <h1>Product Pick</h1>
+    <h1>NEW FALL COLLECTION</h1>
+    <h2> LOGIN TO OUR EXCLUSIVE CLUB OFFERS</h2>
+    <form class="log-user">
+      <label for= "name" type="text">Name</label>
+      <input required minlength="2" type="text" id="name" placeholder="Name and Last Name"><br><br>
+       <label for="email">E-mail</label>
+       <input required  type="email" id="email" name="email" placeholder="Please confirm your email"><br>
+
+       <button class="btn" type="submit" name="submit" value="sub">Submit</button>
+    </form>
+  
   </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+   <!--END HEADER-->
+
+<main>
+    <!--  
+          a valid name. 
+          require a valid email.
+          require one colour from a defined set of 3 colours.
+          require one size from a defined set of 6 sizes.
+          require one date from a constrained date range.-->
+
+        <section  class="product-card">
+          <div class="image-product">
+            <img class="product-img" src="img/jumper1.jpg" alt="jumper cashmere yellow">
+          </div>
+          <section>
+          
+             <!-- MUST HAVE THE same NAME TO allowed only 1 been selected-->
+             
+             <form class="product-features">
+              <p class="product-name">PULLOVER AGHATA</p>
+              <p class="product-price">£19.90</p>
+            <legend>Color:</legend>
+
+             <!--COLURS !!!REMEMBER IMPORTANT THE SAME NAME SO ONLY ONE CAN BE CHOSEN!!!-->
+
+             <label for="pink"><img class="image-jumpers" src="img/pink.jpg" alt="image of a soft pink jumper"> </label>
+             <input type="radio" id="pink" name="color" value="pink" required>
 
+             <label for="yellow"><img class="image-jumpers" src="img/yellow.jpg" alt="image of a soft pink jumper"> </label>
+             <input type="radio" id="yellow" name="color" value="yellow" required>
+
+             <label for="green"><img class="image-jumpers" src="img/green.jpg  " alt="image of a soft pink jumper"> </label>
+             <input type="radio" id="green" name="color" value="green" required>
+           
+
+             <label for="orange"><img class="image-jumpers" src="img/orange.jpg" alt="image of a soft pink jumper"> </label>
+             <input type="radio" id="orange" name="color" value="orange" required>
+
+             <label for="blue"><img class="image-jumpers" src="img/blue.jpg  " alt="image of a soft pink jumper"> </label>
+             <input type="radio" id="blue" name="color" value="blue" required>
+           
+            
+             <!--SIZES !!!REMEMBER IMPORTANT THE SAME NAME SO ONLY ONE CAN BE CHOSEN!!-->
+             <legend>Sizes:</legend>
```

Comment:
(Legend must be first child of fieldset.)

Reply:
also. 
     <fieldset>
              <legend>Sizes:</legend>